pom.xml

```diff
@@ -33,8 +33,11 @@
         <version.maven-compiler-plugin>3.8.1</version.maven-compiler-plugin>
         <version.maven-surefire-plugin>3.0.0-M1</version.maven-surefire-plugin>
         <version.maven-failsafe-plugin>3.0.0-M1</version.maven-failsafe-plugin>
-        <version.liberty-maven-plugin>3.3.2</version.liberty-maven-plugin>
+        <version.liberty-maven-plugin>3.3.4</version.liberty-maven-plugin>
         <liberty.bootstrap.project.name>${project.artifactId}</liberty.bootstrap.project.name>
+        <liberty.var.default.http.port>9080</liberty.var.default.http.port>
+        <liberty.var.default.https.port>9443</liberty.var.default.https.port>
+        <liberty.var.app.context.root>api</liberty.var.app.context.root>
```

Comment:
This isn't the context root.. it's the path.  Don't care much about the var name but don't want to misspeak.  Context Root is controlled by the deployer.. it's just "/"  ...    within the app it's the JAX-RS app path + resource path.